File: drainit-website/app/TitikTersumbat.php
<?php

namespace App;

use GoldSpecDigital\LaravelEloquentUUID\Database\Eloquent\Model;


class TitikTersumbat extends Model
{
    protected $fillable = [
        'id_admin', 'geometry', 'nama_jalan', 'foto','keterangan'
    ];
    protected $spatialFields = [
        'geometry'
    ];
    protected $hidden = [
        'id_admin',
    ];
}

File: app/src/main/java/com/azizapp/test/model/TitikTersumbat.kt
package com.azizapp.test.model


import com.google.gson.annotations.SerializedName

data class TitikTersumbat(
    @SerializedName("foto")
    var foto: String,
    @SerializedName("geometry")
    var geometry: String,
    @SerializedName("id")
    var id: String,
    @SerializedName("keterangan")
    var keterangan: String,
    @SerializedName("nama_jalan")
    var namaJalan: String,
    @SerializedName("status")
    var status: String?
)